query-engine/dmmf/src/tests/setup.rs

```diff
@@ -60,7 +60,7 @@ fn format_diff(old: &str, new: &str) -> String {
     let diff = differ.algorithm(Algorithm::Patience).diff_lines(old, new);
     let mut buf = String::new();
 
-    for (idx, group) in diff.grouped_ops(3).iter().enumerate() {
+    for (idx, group) in diff.grouped_ops(6).iter().enumerate() {
```

Comment:
What does this change mean?

Reply:
That the diff groups that are shown for the snapshots take 6 lines before & after the diff instead of 3